source/_addons/duckdns.markdown

```diff
@@ -50,7 +50,8 @@ Use the following configuration in Home Assistant to use the generated certifica
 
 ```yaml
 http:
-  base_url: https://my-domain.duckdns.org:8123
+  base_url: my-domain.duckdns.org
```

Comment:
This change is incorrect.
It should be a full URI, also see: <https://www.home-assistant.io/integrations/http#base_url>